api.planx.uk/modules/send/email/index.ts

```diff
@@ -34,13 +34,20 @@ export const sendToEmail: SendIntegrationController = async (
     const { email, flow } = await getSessionEmailDetailsById(sessionId);
     const flowName = flow.name;
 
+    // Make application files download magic link
+    const params = new URLSearchParams({
+      email: teamSettings.submissionEmail,
+      localAuthority: localAuthority,
+    });
+    const applicationFilesDownloadLink = `${process.env.API_URL_EXT}/download-application-files/${sessionId}?${params}`;
```

Comment:
```suggestion
    const applicationFilesDownloadLink = `${process.env.API_URL_EXT}/download-application-files/${sessionId}?${params.toString()}`;
```

Reply:
Not tested yet (doing so now) but this flagged as an issue with this approach!

Reply:
Ah ignore this sorry - when templated this parses to a string natively without `.toString()` - neat!

Reply:
Exactly :slightly_smiling_face: